src/components/utils/TextEditor.js

```diff
@@ -0,0 +1,236 @@
+import { Box, FormControl, FormHelperText, makeStyles, Typography } from "@material-ui/core";
+import { FormatBold, FormatItalic, FormatListBulleted, FormatListNumbered, FormatUnderlined } from "@material-ui/icons";
+import { ToggleButton, ToggleButtonGroup } from "@material-ui/lab";
+import Placeholder from "@tiptap/extension-placeholder";
+import Underline from "@tiptap/extension-underline";
+import Heading from "@tiptap/extension-heading";
+import { EditorContent, useEditor } from "@tiptap/react";
+import StarterKit from "@tiptap/starter-kit";
+import React, {  useEffect } from "react";
+import { CreateOfferConstants } from "../Offers/New/CreateOfferUtils";
+import { PropTypes } from "prop-types";
+
+const useStyles = makeStyles((theme) => ({
+    editorToolbar: {
+        marginBottom: theme.spacing(1),
```

Comment:
@DoStini This should fix the alignment problem:
```suggestion
        marginBottom: theme.spacing(1),
        display: "flex",
```